steps/upload-to-moquality/v0.1.0/step.yml

```diff
@@ -0,0 +1,70 @@
+title: Upload to MoQuality
+summary: |
+  Uploads an APK file to MoQuality
+description: |
+  This step logs into your MoQuality account and uploads an APK file to MoQuality. Upon upload, a new version of your app is created with the given APK file.
+website: https://github.com/moquality/bitrise-step-upload-to-moquality
+source_code_url: https://github.com/moquality/bitrise-step-upload-to-moquality
+support_url: https://github.com/moquality/bitrise-step-upload-to-moquality/issues
+published_at: 2018-08-20T10:45:56.686234965-04:00
+source:
+  git: https://github.com/moquality/bitrise-step-upload-to-moquality.git
+  commit: beb3c0152d61c30bb5c756a99815b1ff51637652
```

Comment:
The tag used for versioning the step is invalid for our StepLib. Please remove the `v` from the tag. It should be in a standard semver format like `0.0.0`.

Please do a re-share by deleting the current branch from your fork as well.

Thanks for the PR btw!